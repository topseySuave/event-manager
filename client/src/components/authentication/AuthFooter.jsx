import React, { Component } from 'react';
import { Link } from 'react-router-dom';

/**
 * AuthFooter Class Component
 * */
class AuthFooter extends Component {
  /**
   * @returns { component }
   * */
  render() {
    return (
      <div className="auth__footer footer">
        <div className="container">
          <div className="center-align">
            <span>
              {' '}
              Â© 2017, All rights reserved.{' '}
              <Link to="/" href="/">
                Boots Events Manager
              </Link>
            </span>
          </div>
        </div>
      </div>
    );
  }
}

export default AuthFooter;
