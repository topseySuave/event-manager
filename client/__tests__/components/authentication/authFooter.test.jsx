import React from 'react';
import { shallow } from 'enzyme';
import AuthFooter from
  '../../../src/components/authentication/AuthFooter';

describe('Authentication footer Component', () => {
  const wrapper = shallow(<AuthFooter />);
  test('make a snapshot of the AuthFooter component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
