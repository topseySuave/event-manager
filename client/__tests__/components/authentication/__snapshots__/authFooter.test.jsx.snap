// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication footer Component make a snapshot of the AuthFooter component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AuthFooter />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        <div
          className="center-align"
        >
          <span>
             
            © 2017, All rights reserved.
             
            <Link
              href="/"
              replace={false}
              to="/"
            >
              Boots Events Manager
            </Link>
          </span>
        </div>
      </div>,
      "className": "auth__footer footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="center-align"
        >
          <span>
             
            © 2017, All rights reserved.
             
            <Link
              href="/"
              replace={false}
              to="/"
            >
              Boots Events Manager
            </Link>
          </span>
        </div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <span>
             
            © 2017, All rights reserved.
             
            <Link
              href="/"
              replace={false}
              to="/"
            >
              Boots Events Manager
            </Link>
          </span>,
          "className": "center-align",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              " ",
              "© 2017, All rights reserved.",
              " ",
              <Link
                href="/"
                replace={false}
                to="/"
              >
                Boots Events Manager
              </Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            " ",
            "© 2017, All rights reserved.",
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Boots Events Manager",
                "href": "/",
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": "Boots Events Manager",
              "type": [Function],
            },
          ],
          "type": "span",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <div
            className="center-align"
          >
            <span>
               
              © 2017, All rights reserved.
               
              <Link
                href="/"
                replace={false}
                to="/"
              >
                Boots Events Manager
              </Link>
            </span>
          </div>
        </div>,
        "className": "auth__footer footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="center-align"
          >
            <span>
               
              © 2017, All rights reserved.
               
              <Link
                href="/"
                replace={false}
                to="/"
              >
                Boots Events Manager
              </Link>
            </span>
          </div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <span>
               
              © 2017, All rights reserved.
               
              <Link
                href="/"
                replace={false}
                to="/"
              >
                Boots Events Manager
              </Link>
            </span>,
            "className": "center-align",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " ",
                "© 2017, All rights reserved.",
                " ",
                <Link
                  href="/"
                  replace={false}
                  to="/"
                >
                  Boots Events Manager
                </Link>,
              ],
            },
            "ref": null,
            "rendered": Array [
              " ",
              "© 2017, All rights reserved.",
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Boots Events Manager",
                  "href": "/",
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Boots Events Manager",
                "type": [Function],
              },
            ],
            "type": "span",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
